src/main/java/duke/parser/Parser.java

```diff
@@ -0,0 +1,75 @@
+package duke.parser;
+
+import static duke.util.Keyword.KEYWORD_BYE;
+import static duke.util.Keyword.KEYWORD_DEADLINE;
+import static duke.util.Keyword.KEYWORD_DELETE;
+import static duke.util.Keyword.KEYWORD_DONE;
+import static duke.util.Keyword.KEYWORD_EVENT;
+import static duke.util.Keyword.KEYWORD_FIND;
+import static duke.util.Keyword.KEYWORD_HELP;
+import static duke.util.Keyword.KEYWORD_LIST;
+import static duke.util.Keyword.KEYWORD_MULTIPLE_SPACE;
+import static duke.util.Keyword.KEYWORD_ONE_SPACE;
+import static duke.util.Keyword.KEYWORD_REMIND;
+import static duke.util.Keyword.KEYWORD_TODO;
+
+import duke.commands.AddCommand;
+import duke.commands.ByeCommand;
+import duke.commands.Command;
+import duke.commands.DeleteCommand;
+import duke.commands.DoneCommand;
+import duke.commands.FindCommand;
+import duke.commands.HelpCommand;
+import duke.commands.ListCommand;
+import duke.commands.ReminderCommand;
+import duke.commands.UnknownCommand;
+
+/**
+ * Class that simulates reading the user's input and making sense of it.
+ */
+public class Parser {
+
+    /**
+     * Making sense of the user's input and outputting the relevant commands to the input.
```

Comment:
I think a better way to write this javadocs is "Makes sense of..."

Reply:
thanks!